src/vivarium_public_health/risks/distributions.py

```diff
@@ -266,7 +266,7 @@ def ppf(self, quantiles: pd.Series) -> pd.Series:
 class PolytomousDistribution(RiskExposureDistribution):
     @property
     def categories(self) -> List[str]:
-        return self.lookup_tables["exposure"].value_columns
+        return sorted(self.lookup_tables["exposure"].value_columns)
```

Comment:
Just to check - is it possible to sort this earlier?

Reply:
I believe this is where we want to sort this because it is specific to this component but we could find a spot to check this in lookup table creation.

Reply:
I still think the sorting should actually happen when/where you actually need it sorted, i.e. right at the `.cumsum`. That said, Rajan said just adding a comment is fine so I'll approve

Reply:
I actually think the sorting should happen in `get_exposure_value_columns` in line 292, since that is where we originally lose the ordering, so if you looked at the LookupTable data in the debugger the columns would be ordered properly. That being said, it isn't a huge deal either way.